ui/js/settings.js

```diff
@@ -1,8 +1,13 @@
 /* globals confirm, wp_stream, ajaxurl, wp_stream_regenerate_alt_rows */
 jQuery( function( $ ) {
-
-	var keepRecordsIndefinitely = $( '#wp_stream\\[general_keep_records_indefinitely\\]' ),
-	    keepRecordsFor          = $( '#wp_stream_general_records_ttl' ),
+	var network_affix;
+	if ( 'wp_stream_network' === $( 'input[name="option_page"]' ).val() ) {
+		network_affix = '_network_affix';
+	} else {
+		network_affix = '';
+	}
+	var keepRecordsIndefinitely = $( '#wp_stream' + network_affix + '\\[general_keep_records_indefinitely\\]' ),
```

Comment:
I'll check on that, it is not a change I made, I just added the network affix part.